homeassistant/components/sensor/broadlink.py

```diff
@@ -19,7 +19,10 @@
 from homeassistant.util import Throttle
 import homeassistant.helpers.config_validation as cv
 
-REQUIREMENTS = ['broadlink==0.2']
+
+REQUIREMENTS = ['https://github.com/mjg59/python-broadlink/archive'
```

Comment:
We need the updated version of the lib. Maybe @mjg59 can make a new release for Pypi?